### PR TITLE
Added 'Changing scenes manually' section

### PR DESCRIPTION
**Pull request by willnationsdev**
Adding more detailed documentation on the different ways that one can change scenes manually, and the pros and cons therein. I made sure to put the stuff on Hemingway Editor for proofreading before making the PR. Looking forward to your guys' feedback.

### CODE REVIEW

**Comment by akien-mga** [comment]
While that's good documentation, I'm not sure that it should be in the Step by step tutorial which is meant for Godot beginner's first interaction with the engine. The mention of `change_scene_to` is fine, but the rest is probably too advanced material.

**Comment by cbscribe** [comment]
I think tutorials/misc is a good place for it. Agreed it's a bit too complex for the step-by-step section.

**Comment by willnationsdev** [comment]
Updated it to use a tutorials/misc section for the latter half of the changes.

**Comment by willnationsdev** [comment]
Oh, looks like I need to proofread the larger section once more. There are some syntax/content issues that assumed the scene tree stuff existed before it.

File: tutorials/misc/change_scenes_manually.rst [hunk 1]
```diff
@@ -0,0 +1,139 @@
+.. _doc_change_scenes_manually:
+
+Change scenes manually
+========================
```
Comment (akien-mga) [inline tutorials/misc/change_scenes_manually.rst]: Underline doesn't match title (two chars too wide :)).

File: tutorials/misc/change_scenes_manually.rst [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+Sometimes it helps to have more control over how one swaps scenes around.
+As mentioned above, of a :ref:`Viewport <class_Viewport>`'s child nodes
```
Comment (akien-mga) [inline tutorials/misc/change_scenes_manually.rst]: "of" is meant to be removed I guess?

File: tutorials/misc/change_scenes_manually.rst [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+will render to the image it generates. This holds true even for nodes outside
+of the "current" scene. Autoloads fall into this category, but so also do
```
Comment (akien-mga) [inline tutorials/misc/change_scenes_manually.rst]: "so also do" sounds weird - "so do" is enough, no?

File: tutorials/misc/change_scenes_manually.rst [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+scenes which one instances and adds to the tree at runtime:
+
+.. tabs::
+ .. code-tab:: gdscript GDScript
+
+    var simultaneous_scene = preload("res://levels/level2.tscn")
+
+    func _add_a_scene_manually():
+        # this is like autoloading the scene, only
```
Comment (akien-mga) [inline tutorials/misc/change_scenes_manually.rst]: Comments start with a capital letter.

**Comment by willnationsdev** [comment]
Implemented the requested changes @akien-mga .

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**